chain/tests/test_pibd_validation.rs

```diff
@@ -87,7 +87,7 @@ fn test_pibd_chain_validation_impl(is_test_chain: bool, src_root_dir: &str) {
 		println!("BITMAP PMMR NUM_LEAVES: {}", bitmap_mmr_num_leaves);
 
 		// And total size of the bitmap PMMR
-		let bitmap_pmmr_size = pmmr::peaks(bitmap_mmr_num_leaves + 1)
+		let bitmap_pmmr_size = pmmr::peaks(bitmap_mmr_num_leaves)
```

Comment:
I assume this is because the former was written before my 1-based -> 0-based conversion?

Reply:
yes